assets/ExpressionMetadata.json

```diff
@@ -458,6 +458,18 @@
                 }
             ]
         },
+        {
+            "name": "listKeyValue",
+            "expectedUsage": "listKeyValue(resourceName\/resourceIdentifier, apiVersion, KeyLabelObject)",
+            "description": "Returns a value from the App Config store with Key/Label specified in the request. The KeyLabelObject parameters is a JSON object with the two properties \"key\" and \"label\" as required by the values in the store.",
```

Comment:
How about an example?  Some of these are difficult to figure out.  For instance, I'm guessing keyLabelObject has to be a variable/param reference?  Is there a way to create a JSON object directly in TLE?

Reply:
Can we provide json within the json?  Can certainly do that...

Reply:
So, did you want to add something to the description here to clarify usage or leave it alone?

Reply:
I'm fine with it as is...